src/core_plugins/kbn_vislib_vis_types/public/controls/vislib_basic_options.html

```diff
@@ -1,4 +1,15 @@
 <div>
+  <div class="vis-option-item" ng-if="vis.params.showLegend">
```

Comment:
When/how is `vis.params.showLegend` set to true? I wasn't sure of the right way of making this option appear so I had to manually remove this `ng-if` in order to test this with a pie chart.